internal/docker/supported.go

```diff
@@ -25,23 +25,26 @@ func startDbInDockerSupported(dialect string) (cleanup func() error, retURL, con
 	}
 
 	var resource *dockertest.Resource
-	var url string
+	var url, dialect, tag string
+	dialect, tag = splitImage(opt...)
+
 	switch dialect {
 	case "postgres":
 		switch {
 		case os.Getenv("BOUNDARY_TESTING_PG_URL") != "":
 			url = os.Getenv("BOUNDARY_TESTING_PG_URL")
 			return func() error { return nil }, url, "", nil
+
 		default:
-			resource, err = pool.Run("postgres", "11", []string{"POSTGRES_PASSWORD=password", "POSTGRES_DB=boundary"})
+			resource, err = pool.Run(dialect, tag, []string{"POSTGRES_PASSWORD=password", "POSTGRES_DB=boundary"})
 			url = "postgres://postgres:password@localhost:%s?sslmode=disable"
 			if err == nil {
 				url = fmt.Sprintf("postgres://postgres:password@%s/boundary?sslmode=disable", resource.GetHostPort("5432/tcp"))
 			}
 		}
 
 	default:
-		panic(fmt.Sprintf("unknown dialect %q", dialect))
+		fmt.Errorf("unknown dialect %q", dialect)
```

Comment:
Did you mean to assign to `err` here? As it is this will basically be a no-op and won't trigger exiting in the next block.

Reply:
🤦🏻  I meant to revert this, my b